CDCgov/prime-reportstream
# 7785 Remove feature flag for Message Tracker

This PR ...

Enables the Message Tracker feature in the Admin dropdown

Test Steps:
1. Log into RS
2. Select Admin
3. Notice the Message Id Search feature

## Changes
- Removed logic that hid the message tracker behind the feature flag
![Screenshot 2023-03-02 at 11 13 07 AM](https://user-images.githubusercontent.com/102491809/222528551-72c7c7a1-b2be-4a72-9c27-b088261eb960.png)

## Checklist

### Testing


## Linked Issues
- Fixes #7785
